app/tasks/s3.py

```diff
@@ -1,25 +1,34 @@
-from __future__ import \
-    unicode_literals, \
-    absolute_import
+from __future__ import absolute_import, unicode_literals
 
-from . import celery
-from app import db
-from flask import current_app
+import zipfile
 from io import BytesIO
 from os import getenv
-from app.api.services import brief_responses_service
-from app.api.csv import generate_brief_responses_csv
-from app.models import Brief, BriefResponse
-from werkzeug.utils import secure_filename
+
 import boto3
 import botocore
-import zipfile
+from flask import current_app, render_template
+from jinja2 import Environment, PackageLoader, select_autoescape
+from werkzeug.utils import secure_filename
+
+from app import db
+from app.api.csv import generate_brief_responses_csv
+from app.api.services import brief_responses_service
+from app.models import Brief, BriefResponse
+from app.api.helpers import prepare_specialist_responses
+
+from . import celery
 
 
 class CreateResponsesZipException(Exception):
     """Raised when the resume zip fails to create."""
 
 
+template_env = Environment(
+    loader=PackageLoader('app.tasks', 'templates'),
+    autoescape=select_autoescape(['html', 'xml', 'md'])
```

Comment:
I've recently discovered that jinja2 does not have any concept of escaping Markdown, so `'md'` here is redundant.